qualtran/bloqs/chemistry/df/common_bitsize.py

```diff
@@ -0,0 +1,18 @@
+#  Copyright 2023 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+
+def get_num_bits_lxi(num_aux, num_xi, num_spin_orb) -> int:
```

Comment:
docstring. Does this need its own file? Should this function be private?

Reply:
I actually deleted it as it's not appropriate (we now read the total number of eigenvalues ~ num_aux*num_xi, and num_xi is fixed by the basis set size).